### ISSUE TITLE
Model performance on VCR val split

### ISSUE THREAD

**Issue by JizeCao** [jiasenlu/vilbert_beta#12]
I download the processed VCR data and achieve Q-->A performance about 72.217 with the shared checkpoint, which is slightly lower than the paper claimed. I have used my own processed data and achieve similar performance on both the fine-tuned checkpoint and the pretrained checkpoint after fine tuning on my processed data.

**Comment by jiasenlu** [comment]
I think 0.1 - 0.2 is due to the randomness. I have trained multiple VCR model for the ensemble, can you try using a different random seed?
